pkgs/applications/science/math/R/default.nix

```diff
@@ -35,6 +35,10 @@ stdenv.mkDerivation (finalAttrs: {
     bison imake which blas lapack curl tcl tk jdk tzdata
   ] ++ lib.optionals stdenv.isDarwin [ Cocoa Foundation libobjc libcxx ];
 
+  prePatch = ''
+    substituteInPlace src/unix/sys-unix.c --replace "/bin/sh" "${lib.makeBinPath [ bash ]}/bash"
```

Comment:
Any reason not to use `${lib.getBin bash}/bin/bash` here? (In the sense that semantically `makeBinPath` is for a `:`-separated list)